static/js/containers/pages/login/LoginForgotPasswordPage.js

```diff
@@ -95,50 +98,63 @@ export class LoginForgotPasswordPage extends React.Component<Props, State> {
-          </div>
           {this.state.isError ? (
-            <div className="auth-card card-shadow auth-form">
-              {getCustomerSupportLayout(true)}
-              <EmailForm onSubmit={this.onSubmit.bind(this)} />
+            <div className="row auth-card card-shadow auth-form">
+              <div className="col-12">
+                <div className="row auth-header">
+                  <h1 className="col-12">Forgot Password</h1>
+                </div>
+                {getCustomerSupportLayout(true)}
+                <EmailForm onSubmit={this.onSubmit.bind(this)} />
+              </div>
             </div>
           ) : this.state.forgotEmailSent ? (
-            <div className="card-shadow confirm-sent-page">
-              <h3 className="text-center">Thank You!</h3>
-              {this.state.text}
-              <p>
-                <b>
-                  If you do NOT receive your password reset email, here's what
-                  to do:
-                </b>
-              </p>
-              <ol>
-                <li>
-                  <b>Wait a few moments.</b> It might take several minutes to
-                  receive your password reset email.
-                </li>
-                <li>
-                  <b>Check your spam folder.</b> It might be there.
-                </li>
-                <li>
-                  <b>Is your email correct?</b> If you made a typo, no problem,
-                  just try to
-                  <Link
-                    to={routes.login.forgot.begin}
-                    onClick={this.resetEmailLinkSent.bind(this)}
-                  >
-                    {` reset your password `}
-                  </Link>
-                  again.
-                </li>
-              </ol>
-              {getCustomerSupportLayout(false)}
+            <div className="row auth-card card-shadow">
+              <div className="col-12">
+                <div className="row auth-header">
+                  <h1 className="col-12">Password reset!</h1>
+                </div>
+                {this.state.text}
+                <p>
+                  <b>
+                    If you do NOT receive your password reset email, here's what
+                    to do:
+                  </b>
+                </p>
+                <ul>
+                  <li>
+                    <b>Wait a few moments.</b> It might take several minutes to
+                    receive your password reset email.
+                  </li>
+                  <li>
+                    <b>Check your spam folder.</b> It might be there.
+                  </li>
+                  <li>
+                    <b>Is your email correct?</b> If you made a typo, no
+                    problem,{" "}
+                    <Link
+                      to={routes.login.forgot.begin}
+                      onClick={this.resetEmailLinkSent.bind(this)}
+                    >
+                      {`create an account`}
```

Comment:
This was only in brackets before so we could add spaces on both sides of the string. You can remove the brackets and backticks, or keep them and remove the `{" "}` spaces on both sides of the `Link`. Same comment applies to other pages in this PR